kratos/CMakeLists.txt

```diff
@@ -146,7 +146,9 @@ set( KRATOS_PYTHON_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/python/add_linear_solvers_to_python.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/python/add_factories_to_python.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/python/add_strategies_to_python.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/python/add_utilities_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/python/add_geometrical_utilities_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/python/add_other_utilities_to_python.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/python/add_variable_utilities_to_python.cpp
```

Comment:
For the sake of consistency, I'd call this one as the class it's adding (`VariableUtils`), which is to say: `add_variable_utils_to_python.cpp`.